converter/output/rita/batch/dates/rita_dates.go

```diff
@@ -23,14 +22,12 @@ import (
 //before being sent to MongoDB. The buffers are flushed when
 //they are full or after a deadline passes for the individual buffer.
 type batchRITAConnDateWriter struct {
```

Comment:
The batch writer has to be activated with a cli flag. 
It sends each connection to a timestamp based database and marks them ready for import with ipfix-rita closes. It does not do any database rotation. It has been updated to support strobes via the rita.DB class.

Largely references to the AutoFlushCollection directly tied to the conn collection have been replaced with calls to rita.DB